pocs/focuser/focuser.py

```diff
@@ -138,7 +138,7 @@ def move_to(self, position):
 
     def move_by(self, increment):
```

Comment:
Is the `increment` variable just a count? Maybe some docstring here since it's in base class now.

Reply:
Yeah, it's just the number of steps to change the focuser position by. Could do with a docstring though. There are a few other places where it's not properly documented too.